### PR TITLE
Add users list view with template 

### LINKED ISSUE #24: Add page with users list
We need a list view that will contain all users added to the system. Display email, first name, last name. Pagination would be nice. For now do not add any search.

### PR DESCRIPTION
**Pull request by KamilSupera**
# Issue
Closes: #24 

# Important
Please, merge first #34

### CODE REVIEW

File: elimar/modules/users/urls.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+from .views import FirstUserSignUpView, UsersList
 
 urlpatterns = [
     path(
         "first-signup/",
         FirstUserSignUpView.as_view(),
         name="first-sign-up",
     ),
+    path(
+        "list/",
```
Comment (kaniak274) [inline elimar/modules/users/urls.py]: I would prefer just "", so the URL will look like "/users/"
```suggestion
        "",
```
Wdyt?

File: elimar/modules/users/fixtures/users_accounts.json [hunk 2]
```diff
@@ -0,0 +1,502 @@
+[
+    {
+      "model": "users.user",
+      "pk": 1,
+      "fields": {
+        "first_name": "John_1",
+        "last_name": "Doe_1",
+        "email": "john@doe_1.com",
+        "password": "test"
```
Comment (kaniak274) [inline elimar/modules/users/fixtures/users_accounts.json]: Shouldn't the password be hashed?

File: elimar/templates/users/users_list.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <span class="current">
+            Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}.
+        </span>
+
+        {% if page_obj.has_next %}
+            <a href="?page={{ page_obj.next_page_number }}">next</a>
+            <a href="?page={{ page_obj.paginator.num_pages }}">last &raquo;</a>
+        {% endif %}
+    </span>
+</div>
```
Comment (kaniak274) [inline elimar/templates/users/users_list.html]: I think we should add some indentation here.

**Review by kaniak274 — CHANGES_REQUESTED** [review]

File: elimar/modules/users/urls.py (hunk 1 above)
Comment (KamilSupera) [reply elimar/modules/users/urls.py]: Ahh sorry, it was late when I did this 😨

File: elimar/modules/users/fixtures/users_accounts.json (hunk 2 above)
Comment (KamilSupera) [reply elimar/modules/users/fixtures/users_accounts.json]: I added those dummy users for tests/work crud actions on users.

Comment (KamilSupera) [reply elimar/modules/users/fixtures/users_accounts.json]: That why I did not hash passwords.

**Review by kaniak274 — APPROVED** [review]

**Pull request merged**